tv-shows/src/app/(shows)/layout.tsx

```diff
@@ -10,7 +11,8 @@ export default function ShowsLayout({
 	return (
 		<>
 			<AuthRedirect />
-			<SidebarNavigation />
+			<Toolbar display={{ lg: 'none' }} />
```

Comment:
Probably the same, but we usually use the Chakra Show/Hide component or even better, the `hideBelow`and `hideFrom` props